src/Card/Card.js

```diff
@@ -1,33 +1,25 @@
-import "./Card.css"
-import WikipediaPage from "../WikipediaPage/WikipediaPage"
-import React, {useState} from 'react';
-import DOMPurify from "dompurify";
-import modifyRelativeUrls from "../hooks/modifyRelativeUrls"
-// import star from ".../Media/star"
+import React, { useState } from 'react';
+import DOMPurify from 'dompurify';
+import './Card.css';
+import modifyRelativeUrls from '../hooks/modifyRelativeUrls';
 
+const Card = ({ snippet }) => {
+  const [showFullContent, setShowFullContent] = useState(false);
 
-const Card = ({ title, snippet }) => {
-    const [showPage,setShowPage] = useState(false)
+  const sanitizedSnippet = DOMPurify.sanitize(modifyRelativeUrls(snippet));
 
-  
-    const sanitizedHtml = DOMPurify.sanitize(modifyRelativeUrls(snippet));
+  const snippetToShow = showFullContent ? sanitizedSnippet : sanitizedSnippet.slice(0, 2500) + '...';
```

Comment:
Ah! So this is how you did this, not too complicated! I really like this feature and I think it adds to a flawless user experience!

Reply:
I see you using that ternary operator, nice job!